src/Components/WebAssembly/testassets/StandaloneApp/Pages/ConfigureRuntime.razor

```diff
@@ -0,0 +1,3 @@
+@page "/configure-runtime"
+
+<div id="environment">@Environment.GetEnvironmentVariable("CONFIGURE_RUNTIME")</div>
```

Comment:
Could you please not add this to `testassets/StandaloneApp`? We use this project as a generic playground during development, not as an actual E2E test site (despite the folder name).

We try not to add any random scenario-specific stuff to this project because it's intended to be roughly the same as the default project template. There isn't anything else custom like this in the project currently.

Instead, the E2E test content should ideally go into one of the projects under `src\Components\test\testassets`, for example `BasicTestApp` - you can add scenario-specific stuff to `src\Components\test\testassets\BasicTestApp\wwwroot\index.html`.

Reply:
Thanks for the explanation! The BasicTestApp currently uses "default" startup mechanism. Is it ok if I change it to the manual one with Blazor.start? (that's the reason I originally didn't put it there)

Reply:
Yes, feel free to do that.

Reply:
Oh, maybe I don't need it..